src/downloader.jl

```diff
@@ -44,6 +44,8 @@ function unique_filename(url)
     return string(hash(url))
 end
 
+file_ending(provider::TileProviders.Provider) = ".some_unknown_image_format"
```

Comment:
urls usually end with the image format after you strip the query

Reply:
what should we do here?

Reply:
Splitext(Geturl(provider, 0,0,0))[2] maybe?

On mobile so capitalization is a bit off.  But if anyone wants to do this feel free to go ahead.

Reply:
You may have to strip the query after `?` first

Reply:
`URIs.URI(url).query/path`, parses it into the parts.